cypress/integration/check_summary/summary-count-after-refresh.spec.js

```diff
@@ -0,0 +1,50 @@
+/// <reference types="cypress" />
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false
+})
+
+describe('Testing summary count after refreshing the page', () => {
+  let enabledAttributes = null;
+
+  it('create three different endpoints', () => {
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+    cy.fixture('data').then((data) => {
+      cy.addEndpoint(data.endpoint1)
+      cy.addEndpoint(data.endpoint2)
+      cy.addEndpoint(data.endpoint3)
+    })
+  })
+  it("get third endpoint's enabled attributes amount and refresh the page", () => {
```

Comment:
In other steps of the test, we just get the count of the enabled summary and assign it to a variable.
This test is just to make sure the values are the same after refreshing the page and checking on other steps doesn't add any value because we have to check if a value equals itself.